### PR TITLE
Check translations in github, check formatting strings, fix

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/simplified_chinese.txt [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ https://wiki.ddnet.tw/
 == https://docs.qq.com/doc/DWGFrV0xPRmVWVkla
 
 The width or height of texture %s is not divisible by 16, which might cause visual bugs.
-== 宽和高无法被16整除，这可能会导致画面异常
+== %s: 宽和高无法被16整除，这可能会导致画面异常
```
Comment (def-) [inline data/languages/simplified_chinese.txt:1180]: @TsFreddie Got a better suggestion?

Comment (TsFreddie) [reply data/languages/simplified_chinese.txt:1180]: 材质 %s 的宽和高不能被16整除，这可能会可能导致画面异常。

Comment (TsFreddie) [reply data/languages/simplified_chinese.txt:1180]: just letting you know it is fixed it in the chinese translation PR

**Comment by def-** [comment]
@TsFreddie There's another problem with Chinese string:

```
Your nickname '%s' is already used (%d points). Do you still want to use it?
== 你使用的昵称在DDNet中有%d分的记录，这可能代表改昵称已经被其他人使用过。确认要使用这个名字吗？
```
Where to add the %s?

**Comment by TsFreddie** [comment]
> @TsFreddie There's another problem with Chinese string:
> 
> ```
> Your nickname '%s' is already used (%d points). Do you still want to use it?
> == 你使用的昵称在DDNet中有%d分的记录，这可能代表改昵称已经被其他人使用过。确认要使用这个名字吗？
> ```
> 
> Where to add the %s?

Oops, missed that.

Simp
`你使用的昵称"%s"在DDNet中有%d分的记录，这可能代表改昵称已经被其他人使用过。确认要使用这个名字吗？`

Trad
`你使用的暱稱"%s"在DDNet中有%d分的記錄，這可能代表改暱稱已經被其他人使用過。確認要使用這個名字嗎？`

**Comment by TsFreddie** [comment]
scratch that ^

Simp
`你使用的昵称"%s"在DDNet中有%d分的记录，这可能代表该昵称已经被其他人使用过。确认要使用这个名字吗？`

Trad
`你使用的暱稱"%s"在DDNet中有%d分的記錄，這可能代表這個暱稱已經被其他人使用過。確認要使用這個名字嗎？`

**Comment by def-** [comment]
Thanks, updated.

**Comment by Jupeyy** [comment]
Don't understand this in detail. But i guess good to go.
bors r+

**Pull request merged**